src/dcos_e2e/backends/_aws/__init__.py

```diff
@@ -73,6 +78,8 @@ def __init__(
 
         .. _Regions and Availability Zones:
             https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html
+        .. _Instance types:
```

Comment:
@adamtheturtle : I'm not familiar with this notation, is it correct like this? Or should `..` only be for the first item?

Reply:
This is correct, I believe.

However, to know, build the documentation locally as per http://dcos-e2e.readthedocs.io/en/latest/contributing.html#documentation and see the built document in the local equivalent of http://dcos-e2e.readthedocs.io/en/latest/aws-backend.html#reference.